render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/StackAlignmentErrorClient.java

```diff
@@ -37,6 +47,11 @@ public static class Parameters extends CommandLineParameters {
 		private String stack;
 		@Parameter(names = "--fileName", description = "Name of file to write pairwise errors to (default: errors_<stack>.json.gz)")
 		private String fileName = null;
+		@Parameter(
+				names = "--calculateResiduals",
+				description = "Calculate root mean squared error for each pair of tiles (default: calculate Preibisch error)",
```

Comment:
We should probably rename the errors to something more expressive at some point. In the long run, it would also be cool to handle the different error measures in a polymorphic way, instead of boolean flags. Nothing to be taken care of right now, but a todo probably wouldn't hurt.

Reply:
makes sense - I'll add a todo